src/GBI.h

```diff
@@ -282,6 +284,10 @@ extern u32 G_MWO_aLIGHT_8, G_MWO_bLIGHT_8;
 #define G_TRI_TXTR_ZBUFF		0xCB	/* texture, zbuff triangle:  11001011 */
 #define G_TRI_SHADE_TXTR_ZBUFF	0xCF	/* shade, txtr, zbuff trngl: 11001111 */
 
+// Tile indices
```

Comment:
These indices have such meaning only for sprite microcode.
In general, loadtile is not necessary 7 and rendertile is not necessary 0.
Please move these defines to S2DEX.cpp